src/Currencies.jl

```diff
@@ -55,7 +60,7 @@ Returns the ISO 4217 name associated with this value
 """
 function name end
 
-currency(S::Symbol) = _currency_data[S][1]
+currency(S::Symbol) = _currencies[S]
```

Comment:
Don't change this, change the code so that _currency_data[S][1] has the type, not an instance.